src/System.Private.CoreLib/shared/System/Globalization/CultureInfo.cs

```diff
@@ -1272,13 +1144,37 @@ public static CultureInfo GetCultureInfo(string name)
                 throw new ArgumentNullException(nameof(name));
             }
 
-            CultureInfo? retval = GetCultureInfoHelper(0, name, null);
-            if (retval == null)
+            name = CultureData.AnsiToLower(name);
+            Dictionary<string, CultureInfo> nameTable = CachedCulturesByName;
+            CultureInfo? result;
+
+            lock (nameTable)
+            {
+                if (nameTable.TryGetValue(name, out result))
+                {
+                    return result;
+                }
+            }
+
+            try
```

Comment:
> try [](start = 11, length = 4)

we can use 

private static CultureInfo? CreateCultureInfoNoThrow(string name, bool useUserOverride)

and avoid the try-catch